src/App.js

```diff
@@ -1,25 +1,55 @@
-import logo from './logo.svg';
+import React, { Component } from 'react';
 import './App.css';
+// import Header from './Header.js';
+import axios from 'axios';
+import Form from 'react-bootstrap/Form'
+import Button from 'react-bootstrap/Button'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+require('dotenv').config();
```

Comment:
We can remove line 8 since we use different syntax in React for accessing our .env file. We use {process.env.REACT_API_KEY} etc. 👍 

I would look up when you can using .env in React 👍